templates/repo/issue/view_content/pull.tmpl

```diff
@@ -152,7 +152,7 @@
 							{{$.i18n.Tr "repo.signing.will_sign" .SigningKey}}
 						</div>
 					{{else if .IsSigned}}
-						<div class="item text yellow">
+						<div class="item text black">
 							<i class="icon unlock grey"></i>
```

Comment:
```suggestion
							<i class="icon unlock"></i>
```
match icon to text color for consistency.

Reply:
(I did not change the whitespace in this suggestion, seems like a GitHub bug.

Reply:
Black icon also in the warning case (when there is protection)?

Reply:
If I just remove the color it will be yellow in case of warning
![bild](https://user-images.githubusercontent.com/161914/72643787-c35c9880-396f-11ea-9ffe-e72be3818673.png)

Reply:
Fine with me I suppose.